integration_tests/suite/helpers/base.py

```diff
@@ -27,7 +27,7 @@ class BaseIntegrationTest(AssetLaunchingTestCase):
 
     @classmethod
     def setUpClass(cls):
-        super().setUpClass()
+        super(BaseIntegrationTest, cls).setUpClass()
```

Comment:
Why ? This should not be needed if the project is already in python3

Reply:
it was failing in the integration tests...